masterlist.yaml

```
@@ -525,5 +525,13 @@ plugins:
       - 'Graphic Herbalism TR Extra.ESP'
 
 #Any dynamic patches (not sure if multipatch is the  only one)
+  - name: 'Merged Objects.esp'
+    group: *dynamicPatchGroup
+    msg:
+      - type: say
+        content:
+          - lang: en
+            text: 'Regenrate after adding or removing mods changing or adding in-game items. Use TES3Merge'
+            url: [https://www.nexusmods.com/morrowind/mods/46870]
```

For links in messages use `[example](https://www.example.com/)`.
Add a newline between separate mods as well.

```suggestion
            text: 'Regenerate after adding or removing mods changing or adding in-game items. Use [TES3Merge](https://www.nexusmods.com/morrowind/mods/46870/)'

```

> Regenerate after adding or removing mods changing or adding in-game items.

Wording could be changed slightly.

> Regenerate after adding or removing mods that change or add in-game items.